### PR TITLE
Bulk upload Students/Teachers using .csv files

### PR DESCRIPTION
**Pull request by therealyingtong**
This is based on #151 which is currently unmerged.

Steps to test: 

1. upload relevant `.csv` files using the "Bulk Upload" button from the "Create New Teacher" / "Create New Student" pages
2. check that the new teachers / students show up 
3. please also test that adding an individual New Student / New Teacher through the form still works too.

The `.csv` files contain confidential information and will not be uploaded here. Please find them in our project Slack channel or DM me to get them.

### CODE REVIEW

File: backend/seeds/00200_nativeLanguages.js
Comment (nondescryptid) [inline backend/seeds/00200_nativeLanguages.js]: hi ying tong! Bengali and Bangla refer to the same language. Bengali is usually referred to as Bangla among Bangladeshi people (i volunteer with TWC2 and a lot of guys refer to bengali as bangla, wikipedia also says so here https://en.wikipedia.org/wiki/Bengali_language)

**Comment by therealyingtong** [comment]
The first commit here https://github.com/TechLadies/irrc-madwish/pull/112/commits/a40e742ad4c8dd60d3b146b8e124e4c898e94e0d will conflict with #106.

I'll rebase this PR after we merge #106.

Edit: Merged and rebased.

File: backend/seeds/00200_nativeLanguages.js
Comment (therealyingtong) [reply backend/seeds/00200_nativeLanguages.js]: thanks! i've removed this diff.

**Review by nondescryptid — APPROVED** [review]
Tested and it works! No issues with the code. The csv upload for students initially did not work, but that's because "NativeLanguage" (column header) in the csv file is not the right header — instead it should be "NativeLanguageString". After changing the header in the csv file to NativeLanguageString, students were successfully added and could be viewed

**Comment by therealyingtong** [comment]
Rebased to resolve conflicts with #128.

@szktm I've included the fixes to the helpers which you have in #138. There should be no / trivial conflicts between this PR and #138, so either one could merge first.

**Comment by nondescryptid** [comment]
Screenshot 2021-03-20 at 6 05 35 PM
@therealyingtong just tried it out, i'm able to create new students and teachers (so the helpers should work) and also change their statuses through the profile page. 

However, there are still issues with bulk uploading for teachers as there's a big space above the Teachers table after uploading. Bulk uploading for students doesn't present this issue. Not sure why it's happening

**Comment by therealyingtong** [comment]
@szktm The gap above the Teachers table was fixed by your PR #138 which we have merged. Please review again.

**Review by nondescryptid — APPROVED** [review]

**Review by nondescryptid — APPROVED** [review]
oops i accidentally re-requested a review. I've tried it out again and bulk upload works for both teachers and students. 

Note that this also needs to have conflicts resolved before merging

**Review by aelbore — APPROVED** [review]
needs to resolve conflicts but the code looks good to me.

**Comment by therealyingtong** [comment]
Conflicts resolved after rebasing on latest master.

**Comment by therealyingtong** [comment]
Edit: fixed in https://github.com/TechLadies/irrc-madwish/pull/112/commits/199271619dc20e73838f0ee02b4539543e59427e.

~~This currently breaks individual "New Student" / "New Teacher" for me. Please hold off on merging.~~

**Pull request merged**
